mlrun/projects/project.py

```diff
@@ -1797,28 +1797,126 @@ def set_model_monitoring_application(
-        update or add a monitoring application to the project & deploy it.
+        update or add a monitoring function to the project.
+
         examples::
-            project.set_model_monitoring_application(application_class_name="MyApp",
-                                                 image="davesh0812/mlrun-api:1.5.0", name="myApp")
-        :param func:                    function object or spec/code url, None refers to current Notebook
-        :param name:                    name of the function (under the project), can be specified with a tag to support
+            project.set_model_monitoring_function(application_class_name="MyApp",
+                                                 image="mlrun/mlrun-api:1.5.0", name="myApp")
+
+        :param func:                    Function object or spec/code url, None refers to current Notebook
+        :param name:                    Name of the function (under the project), can be specified with a tag to support
                                         versions (e.g. myfunc:v1)
-                                        default: job
-        :param image:                   docker image to be used, can also be specified in
+                                        Default: job
+        :param image:                   Docker image to be used, can also be specified in
                                         the function object/yaml
-        :param handler:                 default function handler to invoke (can only be set with .py/.ipynb files)
-        :param with_repo:               add (clone) the current repo to the build source
-        :param tag:                     function version tag (none for 'latest', can only be set with .py/.ipynb files)
+        :param handler:                 Default function handler to invoke (can only be set with .py/.ipynb files)
+        :param with_repo:               Add (clone) the current repo to the build source
+        :param tag:                     Function version tag (none for 'latest', can only be set with .py/.ipynb files)
                                         if tag is specified and name is empty, the function key (under the project)
                                         will be enriched with the tag value. (i.e. 'function-name:tag')
-        :param requirements:            a list of python packages
-        :param requirements_file:       path to a python requirements file
+        :param requirements:            A list of python packages
+        :param requirements_file:       Path to a python requirements file
         :param application_class:       Name or an Instance of a class that implementing the monitoring application.
         :param application_kwargs:      Additional keyword arguments to be passed to the
                                         monitoring application's constructor.
         """
 
+        function_object: RemoteRuntime = None
+        (
+            resolved_function_name,
+            function_object,
```

Comment:
```suggestion
        (
            resolved_function_name,
            function_object: RemoteRuntime,
```

Reply:
@alonmr ,Are you sure it is a valid syntax?